streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -127,4 +141,56 @@ GeneralConfiguration getEventsToFhirConfig(String fileName) throws IOException {
 			return generalConfiguration;
 		}
 	}
+	
+	private String getUuid(String parentTable, String parentForeignKey, String childPrimaryKeyValue, Map payload)
+	        throws PropertyVetoException {
+		
+		final String USER_NAME = "root";
+		final String PASS_WORD = "debezium";
+		
+		comboPooledDataSource = new ComboPooledDataSource();
+		comboPooledDataSource.setDriverClass(JDBC_DRIVER);
+		comboPooledDataSource.setJdbcUrl(DB_URL);
+		comboPooledDataSource.setUser(USER_NAME);
+		comboPooledDataSource.setPassword(PASS_WORD);
+		
+		Connection conn = null;
+		Statement stmt = null;
+		String uuidResultFromSql = null;
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
@gitcliff  , can you first refactor this , place it in the in the commons submodule
make one reusable connection method in this [class](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/master/batch/src/main/java/org/openmrs/analytics/JdbcFetchUtil.java#L150)